cdhweb/pages/exodus.py

```diff
@@ -88,7 +99,83 @@ def exodize_attachments(mezzanine_page, wagtail_page):
         logging.debug("attachments for %s: %s" %
                       (wagtail_page, attachments_field))
         wagtail_page.attachments = attachments_field
-        wagtail_page.save()
+        wagtail_page.save(log_action=False)
+    return wagtail_page
+
+
+def exodize_history(mezzanine_page, wagtail_page):
+    """Create log entries for new wagtail page corresponding to old page.
+
+    Also adds a log entry noting that the page was exodized.
+    """
+    # make sure the page has only one real creation event; we want to use
+    # the date it was created in mezzanine
+    creation = PageLogEntry.objects.get(page=wagtail_page,
+                                        action="wagtail.create")
+    creation.delete()
+
+    # get all the old admin log entries for this page. in some cases (seems to
+    # happen for regular content pages) there will be none; in this case we
+    # will end up creating a new revision with a timestamp of right now, and
+    # the only log entry will be marking the exodus
```

Comment:
Is it possible that you're looking at the wrong object? i.e. the log entries are associated with the base page object and not the content page?
(Could also be possible they don't exist!)